src/test/Assets/TestProjects/ComLibraryConflictingGuid/ComLibraryConflictingGuid.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Runtime.InteropServices;
+
+namespace ComLibrary
+{
+    [ComVisible(true)]
+    [Guid("cc6e9910-18d5-484a-a2d2-fa8910fd0261")]
```

Comment:
Please comment on these classes that the guid should be the same. I realize the name of the class is indicating this.